Dockerfile.sle15

```diff
@@ -1,4 +1,4 @@
-FROM registry.suse.de/devel/pubcloud/productionservices/pint/containers/pint/pint-server-lambda-function:latest
+FROM registry.suse.de/home/chajain/devel/pubcloud/productionservices/pint/containers/pint/pint-server-lambda-function:latest
```

Comment:
You still have this using your personal build area path, rather than the standard build area path, as should be the case in a change that is being submitted to the code base.

Reply:
Sorry. I thought I fixed it. Let me do that again